R/dependencies.R

```diff
@@ -144,7 +144,7 @@ dirDependencies <- function(dir) {
     R_files <- grep(ignoredDirRegex, R_files, invert = TRUE, value = TRUE)
   }
 
-  if (opts$fileDependencies.evaluate) {
```

Comment:
What about `if (identical(getOption("packrat.dependency.discovery.enabled"), FALSE))`? IIUC this disables all of the regular dependency discovery done by Packrat.

Reply:
That looks better.  What about `if (!identical(getOption("packrat.dependency.discovery.disabled"), TRUE))`?  This would allow the default case where the variable is not defined to process the dependencies- if I understand things correctly.

Reply:
Sounds great!